WordPress/openverse
# Create sensitive and deleted media models for decisions

## Fixes



Fixes #4513 by @krysal 

## Description



This PR ensures that `SensitiveImage`/`SensitiveAudio` and `DeletedImage`/`DeletedAudio` models are created for every decision.

## Testing Instructions


0. Create a report for an image.
1. Take a deindexed-sensitive or deindexed-copyright action on the report.
   - You should see a new `DeletedImage` object.
   - The image should be deleted.
   - The report should be resolved with a linked decision.
2. Create a report for an image.
3. Take a marked-sensitive action on the report.
   - You should see a new `SenstiveMedia` object.
   - The image should be marked as sensitive.
   - The report should be resolved with a linked decision.

Repeat these steps for audio.

## Checklist


      PRs) or the media properties generator (`./ov just catalog/generate-docs media-props`
      for the catalog or `./ov just api/generate-docs` for the API) where applicable.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
